app/helpers/projects_helper.rb

```diff
@@ -4,6 +4,10 @@ def estimated(story)
     @estimate_id.present?
   end
 
+  def is_locked?(model)
```

Comment:
Instead of using a helper could we use:
```
<% if policy(@post).update? %>
```
or the modal directly?
```
project.locked?
```

Reply:
Here is one or the reasons I prefer to avoid helpers: https://thepugautomatic.com/2012/08/helpers/

Reply:
Hey @fbuys, we will update it as you requested.

But I'd like to mention that I disagreed a little bit, I truly believe that the best way is doing things like the post explains but how often do we disabled global helpers in rails? rails will load helpers anyway.

Also having in as `<% if policy(@post).update? %>` will tend to add more stuff there in the future because will be the easy way `<% if policy(@post).update? && foo && bar %>`

Reply:
We decided to keep this helper for the purpose of refactoring and encapsulating the logic in case of needing to update anything, it is easier to refer to this one method than every where this helper is used.